nistats/design_matrix.py

```diff
@@ -443,3 +443,31 @@ def plot_design_matrix(design_matrix, rescale=True, ax=None):
     ax.set_xticks(range(len(names)))
     ax.set_xticklabels(names, rotation=60, ha='right')
     return ax
+
+
+def _create_second_level_design(maps_table, confounds=None):
+    """Infers a second level design from a maps table.
+
+    The maps table is a pandas DataFrame with at least columns 'map_name' and
+    'subject_id', confounds is also a pandas DataFrame with at least two
+    columns 'subject_id' and one confounder.
+    """
```

Comment:
Also the `confounder` deserves clarification: conceptually, what is the conceptual difference between the confounder and the `maps_table` ?